nikola/nikola.py

```diff
@@ -351,6 +351,14 @@
     }
 }
 
+# Mapping old pre-taxonomy plugin names to new post-taxonomy plugin names
+PLUGIN_NAME_MAP = {
```

Comment:
How about `TAXONOMY_COMPATIBILITY_PLUGIN_NAME_MAP`? That also emphasizes that this is about compatibility.